build.gradle.kts

```diff
@@ -66,15 +67,6 @@ repositories {
     mavenCentral()
 }
 
-tasks.wrapper {
```

Comment:
now this task is embedded into gradle, so no need for a custom one

Reply:
Maybe worth adding this clarification in the final commit message in case people in the future wonder why it was removed.

Reply:
yep, sure, I also updated howto page with steps to upgrade gradle

Reply:
Awesome, thanks!